docs/resource/cloud-provider/README.md

```diff
@@ -1,5 +1,5 @@
 <!--- Hugo front matter used to generate the website version of this page:
-linkTitle: Resource Cloud Provider
+linkTitle: Resource cloud provider
```

Comment:
and below `# Cloud Provider Resource Semantic Conventions`

```suggestion
linkTitle: Cloud provider resources
```

Reply:
If you want the `linkTitle` to be

- Cloud provider resources
 
Then how about this as the title?

> ### Semantic Conventions for Cloud Provider Resources

Reply:
👍

Reply:
Fixed

Reply:
Actually, I've updated the title as agreed. And set the `likeTitle` to "Cloud provider" since this page under the "Resource" section. This is consistent with other pages. Thanks for catching the original error here.

Reply:
👍

Reply:
I'd like to wrap this work up soon, but if we look at other pages in the [Resource](https://github.com/open-telemetry/semantic-conventions/tree/main/docs/resource) section, most have a simple and short title, such as: "Kubernetes", "Webengine", etc. That is, no "Semantic Conventions for ...". Shall we be consistent here? So I'd revert my changes so that the title is just "Cloud Provider"?

Reply:
@trask @lmolkova waiting for your feedback on my previous comment. Otherwise this PR is good to go.

Reply:
> Shall we be consistent here? So I'd revert my changes so that the title is just "Cloud Provider" (or "Cloud Providers")?

sounds good

Reply:
Ok, then we're all set!